packages/core/package.json

```diff
@@ -2190,6 +2208,12 @@
             ]
         },
         "commands": [
+            {
```

Comment:
Is this the name for the task/command? We should confirm the name of it - I was wondering if we should call it `Policy Validator` to keep it aligned with our branding with other tools?

Reply:
The name will be `IAM Policy Checks`